src/components/layout/navbar-items.tsx

```diff
@@ -34,18 +52,76 @@ export const NavbarItems: FC = () => {
                 className={styles.navLinkHover}
             />
 
-            <Group justify="left" pl="xs" c="white">
+            {orgSlug && (
+                <Protect role="org:admin">
+                    <NavLink
+                        label="Admin"
+                        leftSection={<Gear />}
+                        onClick={() => {
+                            if (!pathname.startsWith(orgAdminBaseUrl)) {
+                                setIsAdminMenuOpen((prev) => !prev)
+                            }
+                            // If pathname.startsWith(orgAdminBaseUrl) is true, do nothing here.
+                            // The menu is already open (due to useEffect) and should stay open.
+                        }}
+                        active={false}
+                        opened={isAdminMenuOpen}
+                        c="white"
+                        className={styles.navLinkHover}
+                        rightSection={<></>}
+                    >
+                        <NavLink
+                            label="Manage Team"
+                            leftSection={<UsersThree size={20} />}
+                            component={Link}
+                            href={`${orgAdminBaseUrl}/users`}
+                            active={pathname === `${orgAdminBaseUrl}/users`}
+                            c="white"
+                            color="blue.7"
+                            variant="filled"
+                            className={styles.navLinkHover}
+                            pl="xl"
+                        />
+                        {/* TODO: re-add if we have a org admin settings page */}
+                        {/* <NavLink
+                            label="Settings"
+                            leftSection={<Sliders size={20} />}
+                            component={Link}
+                            href={`${orgAdminBaseUrl}/settings`}
+                            active={pathname === `${orgAdminBaseUrl}/settings`}
+                            c="white"
+                            color="blue.7"
+                            variant="filled"
+                            className={styles.navLinkHover}
+                            pl="xl"
+                        /> */}
+                    </NavLink>
+                </Protect>
+            )}
```

Comment:
this is too much code to put inside a jsx `{}` conditional block.  please extract it to an AdminNavLink (or something named like that) component as previously suggested, then the

Reply:
okay, putting it into a component